mkdocs/docs/supplemental-tools.md

```diff
@@ -0,0 +1,46 @@
+# Supplemental Tools
+
+Tank includes several external/supplemental tools that can be used in conjunction with it for a comprehensive load testing experience.
+
+??? tip "Tip: Navigating to the Tools page"
+    Clicking the `Tools` tab takes the user to the main screen, which has links to `downloads` (AND) pages for launching the tools as `java applets`.
+
+There are two sections of links for the tools, `Applets` and `Downlaodable` Tools.
+
+<figure markdown>
+  ![Tools Main page](https://tank.perf.a.intuit.com/docs/html_single/images/tools/tools_1.png){ width="500" }
+  <figcaption>Tools Main Page</figcaption>
+</figure>
+
+## Java "Applet" Tools
+Applets are rich clients that are run using `Java Applet` technology.   
+They access system resources, and so they are signed using a `self-signed certificate`. This would mean you will be prompted to trust the applet when you launch it.
+
+<figure markdown>
+  !["Applets": Trust certs](https://tank.perf.a.intuit.com/docs/html_single/images/tools/tools_2.png){ width="500" }
```

Comment:
Please use the public domain image content

Reply:
Thanks for the comments Athithya. I will address this.